content/project/teams/hosting.adoc

```diff
@@ -0,0 +1,85 @@
+---
+layout: project
+title: "Hosting Team"
+section: projects
+tags:
+- hosting
+- plugin-site
+links:
+  mailinglist: "/mailing-lists/#jenkinsci-dev-googlegroups-com"
+---
+
+Jenkins Hosting Team handles various requests related to hosting of open-source plugins and other components. (forking and transferring plugins, managing permissions and update centers, etc.).
+There are multiple contributors involved in this activity on regular basis.
```

Comment:
```suggestion
There are multiple contributors involved in this activity on a regular basis.
```

Reply:
Apparently it was spelling which suggested removing the article there :) I also think this is more correct